hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestSchemaEvolution.java

```diff
@@ -595,16 +595,16 @@ private ExpectedResult(String[] evolvedRows, String[] rowsWithMeta, String[] row
           "+I[Alice, 90000.9, unknown, +I[9, s9, 99, t9, drop_add9, 9], {Alice=9999.99}, [9999.0, 9999.0], {Alice=+I[9, s9, 99, t9, drop_add9, 9]}, [+I[9, s9, , 9]], +I[9, 9], [9], {k9=v9}]",
       },
       new String[] {
-          "+I[uuid:id0, Indica, null, 12, null, {Indica=1212.0}, [12.0], null, [+I[0, s0, , 0]], null, null, null]",
```

Comment:
This does not seem correct to me, I thought we should be following the encoding  of `fieldName:fieldValue`, as what is mentioned in the PR description.

Reply:
for single record key and single partition field, we use simple key gen, so no column name encoding here.